.github/workflows/linters.yaml

```diff
@@ -17,6 +17,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install requirements
+        run: ansible-galaxy collection install -r requirements.yml
```

Comment:
I think it should be enough to put the dependency to the `galaxy.yml`, and not have this extra step.